src/locale/de.json

```diff
@@ -779,7 +779,8 @@
       "Notes": "Notizen",
       "PercentComplete": "% Abgeschlossen",
       "Perks": "Perks",
-      "PerksMods": "Perks, Mods & Shader",
+      "Mods": "Mods",
+      "Shaders": "Shaders",
```

Comment:
Please don't edit any localization files other than `i18n.json` (after which you should run `yarn i18n` which will update `en.json`, so only these two should be committed)

Reply:
Will do!